docs/overview.rst

```diff
@@ -43,14 +43,14 @@ Use cases
 - **A generic Web database**: JSON store for mobile and Web apps, games, or IoT...
 - **Quickly prototype frontend applications**: don't lose time with server stuff,
   the backend for your next single page app is already there.
-- **Applications as static files**: just host your apps on GitHub pages, your storage backend
+- **Applications as static files**: just host your apps on GitHub pages, your backend storage
   is elsewhere!
 - **Offline-first applications**: data can also be stored locally and published later.
-- **Build collaborative applications** with real time updates and fine-grained permissions.
+- **Build collaborative applications** with real-time updates and fine-grained permissions.
 - **Synchronise application data** between different devices.
-- **Content delivery**: manage remote content or configuration for your apps via an administration UI
-- **Data collection**: easily collect structured data from surveys, forms, analytics.
-- **Store encrypted data** at a location users can control, ensuring better privacy and security.
+- **Content delivery**: manage remote content or configuration for your apps via an administration UI.
+- **Data collection**: easily collect structured data from surveys, forms, and analytics.
+- **Store encrypted data** at a location that users can control, ensuring better privacy and security.
```

Comment:
This is also a matter of style. Personally I feel like it reads better without the that.